internal/hcp/registry/types.bucket.go

```diff
@@ -599,6 +603,32 @@ type NotAHCPArtifactError struct {
 	error
 }
 
+// AddMetadataToBuild adds metadata to a build in the HCP Packer registry.
+func (bucket *Bucket) AddMetadataToBuild(
+	ctx context.Context, buildName string, metadata packer.BuildMetadata,
+) error {
+	buildToUpdate, err := bucket.Version.Build(buildName)
+	if err != nil {
+		return err
+	}
+
+	preparedMetadata := make(map[string]interface{})
+	preparedMetadata["version"] = metadata.PackerVersion
```

Comment:
I may be wrong, but shouldn't this metadata be under a `packer` key? I was under the impression that we had this requirement last I checked the code on HCP

Reply:
Nevermind, looks like this is an attribute in the structure for the call, no need to wrap the data here!